<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>767</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>767</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="SongList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="track_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>251</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="mute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/img/on_volume.png</normaloff>:/img/on_volume.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Volume">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>385</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="current_duration_ms">
       <property name="text">
        <string>current_duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Duration">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="duration_ms">
       <property name="text">
        <string>duration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>150</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/img/back.png</normaloff>:/img/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Play">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="Shuffle">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repeat">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="Remove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>35</horstretch>
         <verstretch>35</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Add">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
